Formula/z/zigmod.rb

```diff
@@ -0,0 +1,61 @@
+class Zigmod < Formula
+  desc "Package manager for the Zig programming language"
+  homepage "https://nektro.github.io/zigmod/"
+  url "https://github.com/nektro/zigmod/archive/refs/tags/r89.tar.gz"
+  sha256 "3c53d2c50cfc6d0ecc7d1f326e84f11b9d3ffb4c35bc2ad73ec9c4b4f924eec5"
+  license "MIT"
+
+  livecheck do
+    url :stable
+    regex(/^r(\d+)$/i)
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "zig"
+
+  def install
+    # Fix illegal instruction errors when using bottles on older CPUs.
+    # https://github.com/Homebrew/homebrew-core/issues/92282
+    cpu = case Hardware.oldest_cpu
+    when :arm_vortex_tempest then "apple_m1" # See `zig targets`.
+    else Hardware.oldest_cpu
+    end
+
+    args = %W[
+      --prefix #{prefix}
+      -Dtag=#{version}
+      -Dmode=ReleaseSafe
+      -Dstrip=true
+    ]
```

Comment:
Do we need a `*std_zig_args`?

Reply:
for now, I think it is fine, zig is not stable yet (not 1.0.0), and then once it is stable, we can do the zig_std_args like other languages.

Reply:
some people use stdBuildOptions as `optimize`, while zigmods looks like just use `mode`